src/components/Login.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import { toast, ToastContainer } from 'react-toastify'
+import fire from '../config'
+
+function Login () {
+  const history = useHistory()
+
+  const [user, setUser] = useState({
+    email: '',
+    password: ''
+  })
+
+  const { email, password } = user
+
+  const handleChange = (text) => (e) => {
+    setUser({ ...user, [text]: e.target.value })
+  }
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    if (email && password) {
+      fire.auth().signInWithEmailAndPassword(email, password)
+        .then((success) => {
+          setUser({
+            ...user,
+            email: '',
+            password: ''
+          })
+          console.log('Signed in')
+          history.push('/')
+        })
+        .catch((err) => {
+          setUser({
+            ...user,
+            email: '',
+            password: ''
+          })
+          console.log(err)
+          toast.error('You are not registered. Try registering first')
```

Comment:
try getting the message from the response, there can be other reasons for failure like timeouts or any internal server errors

Reply:
Done